components/director/internal/domain/oauth20/service.go

```diff
@@ -195,6 +248,54 @@ func (s *service) doRequest(ctx context.Context, method string, endpoint string,
 	return resp, closeBodyFn, nil
 }
 
+func (s *service) clientsFromHydra(ctx context.Context, endpoint string) ([]Client, error) {
```

Comment:
Could we use hydra sdk [client](https://github.com/ory/hydra/blob/fe8fdc5f6d1ca1ebe16a5241cd06a32d249a4362/docs/versioned_docs/version-v1.8/sdk/go.md), instead of custom code?